### PR TITLE
[#171407708] Deploy eks

### PR DESCRIPTION
**Pull request by DanielVZ96**
[Kibana proxy to EKS](https://www.pivotaltracker.com/story/show/171407708)

# Test
[Check the kibana endpoint here please :)](https://kibana-deploy-eks.eks.admetricks.net/_plugin/kibana/app/kibana)

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -2,5 +2,3 @@ FROM golang:1.7.4-onbuild
 MAINTAINER Dev Admetricks <dev@admetricks.com>
 
 EXPOSE 8080
-
-ENV GIT_SHA fabecdac496d357c9bd366799e580c444286cff0
```
Comment (DanielVZ96) [inline Dockerfile:6]: Not used

**Review by edmolten — APPROVED** [review]

File: k8s.tpl.yml [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+      - name: ${APP_NAME}--proxy-container
+        image: ${IMAGE_NAME}
+        command: ["go-wrapper", "run"]
+        ports:
+        - name: http
+          containerPort: 8080
+          protocol: TCP
+        envFrom:
+          - configMapRef:
+              name: ${APP_NAME}--env
```
Comment (vitorohe) [inline k8s.tpl.yml:68]: no limits?

Comment (Jkaiser001) [reply k8s.tpl.yml:68]: 👍

**Review by vitorohe — APPROVED** [review]
LGTM

File: k8s.tpl.yml (hunk 2 above)
Comment (Jkaiser001) [reply k8s.tpl.yml:68]: fix

**Pull request merged**